array.rb

```diff
@@ -150,36 +150,29 @@ def first n = unspecified = true
   end
 
   # call-seq:
-  #   array.last  -> object or nil
-  #   array.last(n) -> new_array
+  #   last  -> last_object or nil
```

Comment:
Nit: indentation issue.

```suggestion
  #  last  -> last_object or nil
```